docs/compiling.md

```diff
@@ -61,7 +61,8 @@ just the `luac.cross` image.  You must first ensure that the following option in
 `app/include/user_config.h` is matched to your target configuration:
 
 ```c
-//#define LUA_NUMBER_INTEGRAL       // uncomment if you want an integer build
+//#define LUA_NUMBER_INTEGRAL       // uncomment if you want an integer build (LUA5.1)
```

Comment:
maybe use Lua 5.1 or LUA51 but not a mixture of both.
Might also make sense to align these lines with the actual content of `app/include/user_config.h`

Reply:
Done